source/extensions/extensions_build_config.bzl

```diff
@@ -306,6 +306,13 @@ EXTENSIONS = {
 
     "envoy.retry_priorities.previous_priorities":       "//source/extensions/retry/priority/previous_priorities:config",
 
+    #
+    # Retry admission control
+    #
+
+    "envoy.retry_admission_control.static_limits": "//source/extensions/retry/admission_control/static_limits:config",
+    "envoy.retry_admission_control.concurrency_budget": "//source/extensions/retry/admission_control/concurrency_budget:config",
```

Comment:
This extension covers the `retry_budget` field by allowing a set percentage of active concurrent tries to be retries.